app/i18n/en/gen.php

```diff
@@ -72,8 +72,8 @@
 		'december' => 'December',
 		'feb' => 'Feb.',
 		'february' => 'February',
-		'format_date' => '%s j\\<\\s\\u\\p\\>S\\<\\/\\s\\u\\p\\> Y',
-		'format_date_hour' => '%s j\\<\\s\\u\\p\\>S\\<\\/\\s\\u\\p\\> Y \\a\\t H\\:i',
+		'format_date' => 'j\\ %s Y',
+		'format_date_hour' => 'j\\ %s Y \\a\\t H\\:i',
```

Comment:
That \\ isn't for anything anymore, is it?
```suggestion
		'format_date' => 'j %s Y',
		'format_date_hour' => 'j %s Y \\a\\t H\\:i',
```

Reply:
hope so... :D